airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_40_11_002__AddSchemaChangeColumnsToConnections.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.instance.configs.migrations;
+
+import java.util.Arrays;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.jooq.DSLContext;
+import org.jooq.EnumType;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class V0_40_11_002__AddSchemaChangeColumnsToConnections extends BaseJavaMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(V0_40_11_002__AddSchemaChangeColumnsToConnections.class);
+
+  @Override
+  public void migrate(final Context context) throws Exception {
+    LOGGER.info("Running migration: {}", this.getClass().getSimpleName());
+
+    final DSLContext ctx = DSL.using(context.getConnection());
+
+    addNonBreakingChangePreferenceEnumTypes(ctx);
+
+    addNotifySchemaChanges(ctx);
+    addNonBreakingChangePreference(ctx);
+    addBreakingChange(ctx);
+  }
+
+  private static void addNonBreakingChangePreferenceEnumTypes(final DSLContext ctx) {
+    ctx.createType(NonBreakingChangePreferenceType.NAME)
+        .asEnum(Arrays.stream(NonBreakingChangePreferenceType.values()).map(NonBreakingChangePreferenceType::getLiteral).toList())
+        .execute();
+  }
+
+  private static void addNotifySchemaChanges(final DSLContext ctx) {
+    ctx.alterTable("connection")
+        .addColumnIfNotExists(DSL.field(
+            "notify_schema_changes",
+            SQLDataType.BOOLEAN.nullable(false).defaultValue(true)))
```

Comment:
I know we want to have it on by default, just to make sure do we want to turn it on automatically for all existing connections?